lib/index.js

```diff
@@ -212,23 +212,29 @@ Latencies
  */
 const listHandler = async function listHandler(options) {
   const listedFuncs = await list();
+  const rawData = Object.keys(listedFuncs).map(key => ({
```

Comment:
Consider mapping over entries (AFAIR we do have `lodash` in here).

Reply:
done